.zuul.yaml

```diff
@@ -22,7 +22,7 @@
 
 - job:
     name: podified-multinode-edpm-deployment-crc-test-operator
-    parent: podified-multinode-edpm-deployment-crc
+    parent: podified-multinode-hci-deployment-crc-1comp-backends
```

Comment:
@ashu-011 I'm just curious. Why can't we use the edpm job?

Reply:
@lpiwowar We can use but I plan to add some more tests later and the problem is current edpm job doesn't setup a cinder backend due to which all the cinder volume related tests won't work so I switched it with hci job which already setup cinder backend with ceph

Reply:
As discussed with @lpiwowar , Renaming part will be done in a separate PR